src/pages/settings/Profile/CustomStatus/StatusPage.tsx

```diff
@@ -74,6 +74,23 @@ function StatusPage({draftStatus, currentUserPersonalDetails}: StatusPageProps)
         return DateUtils.isTimeAtLeastOneMinuteInFuture({dateTimeString: clearAfterTime});
     }, [draftClearAfter, currentUserClearAfter]);
 
+    const navigateBackToPreviousScreenTask = useRef<{
+        then: (
+            onfulfilled?: () => typeof InteractionManager.runAfterInteractions,
+            onrejected?: () => typeof InteractionManager.runAfterInteractions,
+        ) => Promise<typeof InteractionManager.runAfterInteractions>;
+        done: (...args: Array<typeof InteractionManager.runAfterInteractions>) => typeof InteractionManager.runAfterInteractions;
+        cancel: () => void;
+    } | null>(null);
+
+    useEffect(() => {
```

Comment:
@GandalfGwaihir As my proposal, we should cancel the task on the cleanup function of this useEffect

Reply:
@thesahindia The problem is the implement is wrong here we should cancel the task when the component is unmounted.

Reply:
```
useEffect(() => {
    return () => {
        if (!navigateBackToPreviousScreenTask.current) {
            return;
        }

        navigateBackToPreviousScreenTask.current.cancel();
    }
}, [])
```

The change will be like this.

Reply:
@GandalfGwaihir Please run `npm run lint -- --fix` locally before you push the code to fix the lint error.

Reply:
Look good now.